src/Mapper/MethodGroupMapper.cs

```diff
@@ -35,7 +35,11 @@ public MethodGroupClient Map(MethodGroupJv methodGroup)
             var settings = JavaSettings.Instance;
             
             string package = settings.GetPackage(settings.GenerateClientInterfaces ? settings.ImplementationSubpackage : null);
-            string className = methodGroup.Name;
+            string interfaceName = methodGroup.Name;
+            if (ClientModels.Instance.Any(cm => cm.Name == interfaceName)) {
```

Comment:
Without this fix, we have method group names clashing with model names - like "News" model and "News" operations. This will generate "NewsOperations" method group name.